Code/Source/Entity/EntityManager.h

```diff
@@ -36,6 +38,8 @@ namespace RGL
 
         virtual void Update();
 
+        void UpdateSegmentationClass(AZStd::shared_ptr<AZStd::set<AZStd::pair<AZStd::string, uint8_t>>> classTags);
```

Comment:
Consider adding an alias for this reused type :
AZStd::shared_ptr<AZStd::set<AZStd::pair<AZStd::string, uint8_t>>> and 
AZStd::set<AZStd::pair<AZStd::string, uint8_t>>.
I understand that AZStd::set (instead of AZStd::map) is needed due to reflection in o3de?

Reply:
As a fun fact in o3de this is kept as a vector as vector has better gui generation